### PR TITLE
Added pidfile support for Centos 6

### PR DESCRIPTION
**Pull request by moredure**
Centos6 support requires init.d application process to support pidfile creation

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+imunify-nsq (1.0.1-2) stable; urgency=low
+
+  * Added pidfile support
```
Comment (telepenin) [inline debian/changelog]: Could you please describe here why we need support pid file option explicitly

File: imunify-nsq.spec [hunk 2]
```diff
@@ -100,5 +100,7 @@ if [ $1 -eq 0 ] ; then
 fi
 
 %changelog
+* Tue Jan 17 2023 Mikhail Faraponov <mfaraponov@cloudlinux.com> - 1.0.1-2
+- Added pidfile support
```
Comment (telepenin) [inline imunify-nsq.spec]: And here

Comment (moredure) [reply imunify-nsq.spec]: done

File: debian/changelog (hunk 1 above)
Comment (moredure) [reply debian/changelog]: done

File: apps/nsqd/options.go [hunk 3]
```diff
@@ -133,6 +133,7 @@ func nsqdFlagSet(opts *nsqd.Options) *flag.FlagSet {
 	flagSet.String("tcp-address", opts.TCPAddress, "<addr>:<port> to listen on for TCP clients")
 	flagSet.Bool("use-unix-sockets", opts.UseUnixSockets, "Usinx UNIX sockets instead of IP sockets")
 	flagSet.String("wakeup-socket-dir", opts.WakeupSocketDir, "Directory of sockets to wake up the consumer processes")
+	flagSet.String("pidfile", opts.PIDFile, "Pidfile of current process")
```
Comment (telepenin) [inline apps/nsqd/options.go:136]: I suggest to keep context here (why we need pid file). Actually it neede only for legacy and could be removed right after when centos 6 would be unsupported, i.e. I suggest to use line `Pidfile of current process (centos 6 daemonization requirement)`

**Review by telepenin — APPROVED** [review]

**Pull request merged**